update mathworksservicehost protocol KUL

w-lampaert opened the pull request.

w-lampaert committed:
update mathworksservicehost protocol KUL

w-lampaert committed:
update ps cmd and change phrasing

moravveji reviewed (approved):
LGTM.
You may also consider the to take out the `-v` option from the `rm` command, because it just informs you about useless stuff. Instead the `-f` option returns exit code `0` even if the target folder does not exist. Read all these as a personal suggestion.

WouterVanAssche committed:
rebase and update vsc-Rproject documentation

MaximeVdB committed:
Add Leuven Tier-2 to sites with the Desktop app

MaximeVdB committed:
Mention that clipboard support is browser-dependent

MaximeVdB committed:
Mention that Applications menu is sensitive to XDG_DATA_HOME

MaximeVdB committed:
Typo fix + command formatting in OpenSSH tunnel page

MaximeVdB committed:
Add KU Leuven-specific Desktop app info

MaximeVdB committed:
Warn about the planned NoMachine discontinuation

MaximeVdB committed:
Use tabs for KU Leuven-specific OOD Desktop info

MaximeVdB committed:
Rename 'Limitations' section and move XDG_DATA_HOME tip there

MaximeVdB committed:
Move graphics tip to applications section + use tab-set

MaximeVdB committed:
Mention clipboard support in Microsoft Edge & Safari

MaximeVdB committed:
Order KULeuven/UHasselt and VUB tabs alphabetically

lexming committed:
Merge pull request #527 from MaximeVdB/leuven_ood_desktop

Adding KULeuven OOD Desktop info + NoMachine discontinuation

lexming committed:
Merge pull request #525 from MaximeVdB/misc_updates_20251114

Misc updates 20251114

stevenvdb committed:
Update default GPU compute mode for leuven to shared

w-lampaert commented:
This has slipped my mind. @smoors , is this fine for you?

MaximeVdB committed:
Make Genius/wICE description more future-proof

MaximeVdB committed:
Update python-irodsclient install instructions

Mostly to be compatible with how we expect the Leuven module system to be used
and remove the reference to Broadwell CPUs

MaximeVdB committed:
Correct various typos

MaximeVdB committed:
More consistent scheme for memory bw & latency info

Also updating the P100 node counts for Genius (some have already been
decommissioned).

MaximeVdB committed:
Make Thinking CPU mem bw/latency page visible

MaximeVdB committed:
Update KU Leuven OOD interactive shell environment

The same now as in VUB, so we don't need the site tabs for this one

MaximeVdB committed:
Describe I_MPI_HYDRA_BOOTSTRAP_EXEC_EXTRA_ARGS case

See also https://www.osc.edu/resources/technical_support/known_issues/parallel_job_with_intelmpi_hangs

MaximeVdB committed:
Add ipykernel install step to JupyterLab venv section

MaximeVdB committed:
Use correct end dates for Thinking and BrENIAC

Also adding the start dates for completeness

MaximeVdB committed:
Homogenize the Tier-1 and Tier-2 cluster archives

lexming committed:
add specification of ada_gpu partition in Anansi cluster

smoors committed:
Merge pull request #530 from lexming/ada_gpu

add specification of ada_gpu partition in Anansi cluster

Commit:
Merge branch 'master' into feature/globus_VSC_archive

jefscheepers committed:
Merge pull request #505 from hpcleuven/feature/globus_VSC_archive

add Globus collection for VSC archive

MaximeVdB committed:
Merge pull request #529 from MaximeVdB/misc_updates_20251202

Misc updates 20251202

stevenvdb committed:
Merge pull request #528 from hpcleuven/update_default_compute_mode

Update default GPU compute mode for leuven to shared

stevenvdb committed:
Remove job composer from Open OnDemand pages

stevenvdb committed:
Merge pull request #535 from stevenvdb/remove_ood_job_composer

Remove job composer from Open OnDemand pages

RVerschoren committed:
Update index.rst

dependabot[bot] committed:
Bump urllib3 from 2.5.0 to 2.6.3

Bumps [urllib3](https://github.com/urllib3/urllib3) from 2.5.0 to 2.6.3.
- [Release notes](https://github.com/urllib3/urllib3/releases)
- [Changelog](https://github.com/urllib3/urllib3/blob/main/CHANGES.rst)
- [Commits](https://github.com/urllib3/urllib3/compare/2.5.0...2.6.3)

---
updated-dependencies:
- dependency-name: urllib3
  dependency-version: 2.6.3
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

Commit:
add description for VUB OoD Open WebUI app

smoors committed:
Merge pull request #537 from JarneRenders/open-webui

add docs for VUB OoD Open WebUI app

IneMEGAmaxi committed:
Merge pull request #1 from hpcuantwerpen/update_ua

Update UAntwerp

Added: UA accounting, UA portal acces & UA software instruction (easybuild, conda)
- moved UA software instructions to separate page & fixed some typo's

IneMEGAmaxi committed:
Merge branch 'hpcleuven:master' into master

IneMEGAmaxi committed:
Update tier2_hardware.rst to remove testing note

webportal is no longer in testing stage.

smoors committed:
remove broadwell_himem partition for VUB

Commit:
Restrict VUB Open WebUI to anansi cluster only

smoors committed:
Merge pull request #541 from JarneRenders/open-webui

Restrict VUB Open WebUI to anansi cluster only

lexming committed:
Merge pull request #540 from smoors/broadwell_himem

remove broadwell_himem partition for VUB

Commit:
Update Open WebUI entry

IneMEGAmaxi committed:
Apply suggestions from code review

compute/portal/ondemand/index.rst: format grid-item-cards

Co-authored-by: Sam Moors <smoors@users.noreply.github.com>

epauwels committed:
Update tier1_hortense.rst

epauwels committed:
Update tier1_hortense.rst

epauwels committed:
Update tier1_hortense.rst

hajgato committed:
Merge pull request #544 from hpcleuven/epauwels-patch-4-1

Update tier1_hortense.rst

mstfdkmn committed:
Add iron CLI and pam-interactive authentication method for Tier1 Data

mstfdkmn committed:
Fix typo

mstfdkmn committed:
Add hpc cluster note and fix typo

mstfdkmn committed:
Add notes for vsc login users

mstfdkmn committed:
Add explicit instruction for vsc login users

MaximeVdB committed:
Document the IGV-Web OnDemand app

mstfdkmn committed:
Make instruction clear for iCommands authentication

mstfdkmn committed:
More clarification

mstfdkmn committed:
Merge pull request #554 from hpcleuven/feature/Tier1Data_pam-interactive

Add iron CLI and pam-interactive authentication method for Tier1 Data

JarneRenders committed:
Improve readability

Co-authored-by: Sam Moors <smoors@users.noreply.github.com>

kafetzakid committed:
clarify standard Globus transfer as copy

smoors committed:
Merge pull request #556 from hpcleuven/fix/clarify-Globus-transfer-as-copy

clarify standard Globus transfer as copy

smoors committed:
Merge pull request #543 from JarneRenders/open-webui

Update Open WebUI entry

IneMEGAmaxi committed:
layout and extra explanation in generating_keys_windows.rst,
according to reviewer feedback

IneMEGAmaxi committed:
fix Globus UAntwerpen collection name

esdpoort committed:
Update Gent Tier-2 hardware

esdpoort committed:
Update Gent Tier-2 storage quota

esdpoort committed:
Update Tier-1 hortense

WouterVanAssche committed:
Add section on using the project

moravveji committed:
KU Leuven wICE migration update (#559)

MaximeVdB committed:
Merge pull request #555 from MaximeVdB/ood_igv_web

Document the IGV-Web OnDemand app

IneMEGAmaxi committed:
in generating_keys_windows.rst
per reviewer comments:
- order recommended methods
- remove 'legacy' wording
- clarify windows versions supporting OpenSSH
fixes:
- issues with windows filepaths

IneMEGAmaxi committed:
in generating_keys_windows.rst:
- ssh-keygen does not like backward slashes
- command prompt mentions removed: syntax of variables is different
- if the user account is not admin, running Powershell as admin will
  have $HOME or $env:USERPROFILE pointing to a different location;
  so changed to explicit path with <user> placeholder

esdpoort committed:
Correct partition hardware on Tier-1 hortense

moravveji committed:
FileZilla authentication update for KUL (#558)

mselwa committed:
Merge branch 'hpcleuven:master' into master

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update security_measures_20200520.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

smoors committed:
Merge pull request #536 from hpcleuven/dependabot/pip/urllib3-2.6.3

Bump urllib3 from 2.5.0 to 2.6.3

IneMEGAmaxi committed:
Apply suggestions from code review

Co-authored-by: Alex Domingo <alex.domingo.toro@vub.be>

IneMEGAmaxi committed:
adding example of ssh config for windows

lexming committed:
Merge pull request #539 from hpcuantwerpen/feature/openssh_windows

Feature/openssh windows

lexming committed:
update documentation on SSH config to be usable by all VSC sites

lexming committed:
configure caption label for all supported elements

lexming committed:
Merge branch 'master' into ssh-config

lexming committed:
add caption to SSH config file for Windows

lexming committed:
simplify title of ssh agent section in generating keys on Windows

lexming committed:
add caption to SSH config file

smoors committed:
add hopper_gpu partition for brussel

smoors committed:
Update source/brussels/tier2_hardware/hydra.rst

Co-authored-by: wpoely86 <wpoely86@gmail.com>

smoors committed:
add missing links

lexming committed:
Merge pull request #562 from smoors/hopper

add `hopper_gpu` partition for brussel

lexming committed:
Apply suggestions from code review

Co-authored-by: Sam Moors <smoors@users.noreply.github.com>

lexming committed:
hide caption labels in code-blocks

lexming committed:
add warning about needing SSH agent to use proxy host in SSH

avapirev committed:
Update vscode-tunnel.rst to include KUL/UH sites (#499)

smoors committed:
Merge pull request #561 from lexming/ssh-config

update documentation on SSH config to be usable by all VSC sites

smoors committed:
skip unnecessary intermediate step to the ondemand docs

smoors committed:
make suggested changes

smoors committed:
Merge pull request #538 from hpcuantwerpen/master

Update UAntwerp

smoors committed:
merge main

lexming committed:
Merge pull request #565 from smoors/ondemand-direct

skip unnecessary intermediate step to the ondemand docs

hajgato committed:
Merge pull request #557 from esdpoort/update_gent

Update gent

smoors committed:
re-format vscode tunnel section (#564)

mselwa committed:
Update authentication.rst

Remarks from Alex and typo's

mselwa committed:
Update authentication.rst

layout

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

Commit:
Add VUB matlab parallel docs

stevenvdb committed:
Add section about how to use EESSI to the Leuven module system page

moravveji committed:
updates after wICE migration to rocky9 (#568)

stevenvdb committed:
Improve readability of EESSI section

stevenvdb committed:
Recommend to load EESSI/2023.06 instead of default version

mselwa committed:
Update authentication.rst

remarks Jan

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update authentication.rst

mselwa committed:
Update security_measures_20200520.rst

extra firewall info (minimalistic)

mselwa committed:
Update security_measures_20200520.rst

mselwa committed:
Update authentication.rst

moravveji committed:
KUL OOD Matlab version policy (#570)

change KUL Matlab version policy

lexming committed:
Merge pull request #560 from mselwa/mselwa-patch-firewall

patch firewall KU Leuven

lexming committed:
Merge pull request #571 from hpcleuven/patch-firewall

Patch firewall

MaximeVdB committed:
Update planning for Genius decommissioning

stevenvdb committed:
Merge pull request #569 from hpcleuven/add_eessi_section_to_leuven_module_system

Add section about how to use EESSI to the Leuven module system page

MaximeVdB committed:
Update the GIO instructions for Leuven drives

MaximeVdB committed:
Merge pull request #572 from MaximeVdB/genius_decommissioning_update

Update planning for Genius decommissioning

Commit:
Fix style issues

Commit:
Integrate VUB info with KUL info using tabs

Commit:
Add some small fixes

Commit:
Small fix

MaximeVdB committed:
Add note about Genius throttling

MaximeVdB committed:
Merge pull request #574 from MaximeVdB/genius_throttling

Add note about Genius throttling

Commit:
Replace tabs in intro by badges

Commit:
Fix indentation issues

Commit:
Remove salloc mention in VUB part

Commit:
Various changes

Commit:
Swap interactive and batch job and rewrite VUB interactive

smoors committed:
Merge pull request #567 from JarneRenders/matlab

Add VUB matlab parallel docs

stevenvdb committed:
Update EESSI section in Leuven module system page

- Avoid setting $MODULEPATH explicitly by suggesting module use
- Promote EESSI/2025.06 instead of EESSI/2023.06
- Add links to specific EESSI documentation pages

boegel committed:
Merge pull request #577 from stevenvdb/update_eessi_section

Update EESSI section in Leuven module system page

MaximeVdB committed:
Clarify that shared-mem apps need to be run with 1 task

MaximeVdB committed:
Merge pull request #573 from MaximeVdB/update_kuleuven_network_drive_info

Update the GIO instructions for Leuven drives

esdpoort committed:
All partitions to RHEL9

esdpoort committed:
Fix end date Rome partition

hajgato committed:
Merge pull request #582 from esdpoort/update_gent_rhel9

Update gent rhel9

boegel committed:
get rid of RHEL8 debug partitions + fix OS section (RHEL9) for Tier-1 Hortense

hajgato committed:
Merge pull request #583 from boegel/rhel9

get rid of RHEL8 debug partitions + fix OS section (RHEL9) for Tier-1 Hortense

boegel committed:
add note on older software modules that were removed on Tier-1 Hortense RHEL9

hajgato committed:
Merge pull request #584 from boegel/rhel9_rip_pre_2023a

add note on older software modules that were removed on Tier-1 Hortense RHEL9

boegel committed:
fully fix section on debug partition in Tier-1 Hortense

hajgato committed:
Merge pull request #585 from boegel/fix_debug_hortense

fully fix section on debug partition in Tier-1 Hortense

MaximeVdB committed:
Merge pull request #580 from MaximeVdB/slurm_sharedmem_singletask

Clarify that shared-mem apps need to be run with a single task

MaximeVdB committed:
Shift new Tier-2 timeline to Q2 2026

(and remove trailing whitespace)

MaximeVdB committed:
Update the number of H100 GPU nodes

MaximeVdB committed:
Merge pull request #588 from MaximeVdB/leuven_hw_updates_20260323

Leuven tier-2 hardware updates 20260323

moravveji committed:
correct the tab name

lexming committed:
Merge pull request #603 from hpcleuven/join-vsc-groups

ENH: New/Join Group

Commit:
Change sitename from brussels -> to brussel

Commit:
Change brussels sitename for tier2-infrastructure page

JarneRenders committed:
Do not change original url in redirect

Co-authored-by: Sam Moors <smoors@users.noreply.github.com>

Commit:
Remove crash file

smoors committed:
Merge pull request #604 from JarneRenders/brussel-sitename

Change sitename from brussels -> to brussel

Commit:
Add preliminary docs for pilot phase

stevenvdb committed:
[GSSI] Restructure and update software section (#534)

* Restructure the software section

* Update section on Python package management

- Add a paragraph on searching and using packages from modules
- Tweak the paragraph on Python virtual environments
- Move the paragraph on Python packages in conda to the conda-based managers section
- Add a paragraph on vsc-venv based on the UGent documentation
- Add a paragraph on using uv as Python package manager

* Improve stylistics of Python package management sessions

* [GSSI] Add section on software compilation

* Fix blank line and asterisk

Corrected formatting for availability note and added example section.

* update containers section

* Improve section on using modules

- Tweak the page on module system basics
- Remove obsolete reference to leibniz/2019a-experimental module
- Merge two site-specific pages on cluster modules

* Add page on using EESSI

* Mention that cluster modules are used at VUB

* apply minor suggestions for page on using EESSI

Co-authored-by: Kenneth Hoste <kenneth.hoste@ugent.be>

* fix typo + prefer 'heterogeneous' over 'inhomogeneous'

* use code formatting for 'cluster' module

* apply minor suggestions

Co-authored-by: Kenneth Hoste <kenneth.hoste@ugent.be>

* remove duplicate subsection title

* apply minor suggested changes

Co-authored-by: Kenneth Hoste <kenneth.hoste@ugent.be>

* add documentation from GSSI about managing Julia packages

* Address comments on Python package management updates

* Rewrite section on Conda-based managers

With this the docs are more in line with the GSSI training.
This includes switching to a Miniforge module instead of a
(locally installed) Miniconda as the recommended default way
of using Conda.

* Replace UGent/UA/VUB warning with link to Conda pitfalls

* [GSSI] write out page on EasyBuild & Slack

* Duplicate explicit target name: 'here'

* make suggested changes

* remove note about MPI perf

* Refresh page on Perl package management

- Use recent Perl module
- Recommend to load Perl-bundle-CPAN
- Set PERL5LIB instead of relying on loical::lib
- Use VSC environment variables when configuring cpan

* Add page on requesting software

* Use site-agnostic paths

Co-authored-by: Alex Domingo <alex.domingo.toro@vub.be>

* Use site-agnostic paths

Co-authored-by: Alex Domingo <alex.domingo.toro@vub.be>

* Improve the Conda+modules warning

Co-authored-by: Alex Domingo <alex.domingo.toro@vub.be>

* Improve the PyCharm/Conda VSC_HOME warning

Co-authored-by: Alex Domingo <alex.domingo.toro@vub.be>

* Simplify the R devtools Conda instructions

Co-authored-by: Alex Domingo <alex.domingo.toro@vub.be>

* Refine the Conda page structure description

Co-authored-by: Alex Domingo <alex.domingo.toro@vub.be>

* Refine the Miniforge3 location warning

Co-authored-by: Alex Domingo <alex.domingo.toro@vub.be>

* Combine JupyterLab Conda instructions for Leuven/VUB

* update

* update

* format fixes

* update

* fix

* format change

* add documentation from GSSI on R package installation

* improve formatting of vsc-Rproject page

* improve formatting of R devtools page

* fix formatting of IJulia name

* change symlink of ~/.julia from VSC_SCRATCH to VSC_DATA

* clarify inheritance of julia environment from modules

* fix commands to inject Julia packages from module into environment

* conda updates

* fix

* Switch to conda shell func + add note about init

To keep things readable, this required some restructuring

* Use tblite-python instead of icecream

"conda install icecream" will install Python and
for recent Python versions no tblite-python package
may be available, which then leads to errors when
trying to install that package. Requesting to
create a new environment for tblite-python seem to
much of a hassle, so better use tblite-python as an
example throughout.

* Refine the conda run / job example

* No need for site-tabbed job shebang

"#!/bin/bash -l" is OK for all VSC sites

* Address comments about new easybuild_spack page

* Address comments on GSSI R packages update

---------

Co-authored-by: Cintia Willemyns <115622907+WilleBell@users.noreply.github.com>
Co-authored-by: Samuel Moors <sammoors@gmail.com>
Co-authored-by: Kenneth Hoste <kenneth.hoste@ugent.be>
Co-authored-by: Alex Domingo <alex.domingo.toro@vub.be>
Co-authored-by: MaximeVdB <maxime.vandenbossche@kuleuven.be>
Co-authored-by: MaximeVdB <maxime.cp.vandenbossche@gmail.com>
Co-authored-by: Sam Moors <smoors@users.noreply.github.com>

Commit:
Add sofia domain

Commit:
Specify ssh can only happen from uni network

moravveji committed:
Hpc for education (#602)

smoors committed:
add note that it is in pilot phase

smoors committed:
add section on storage

smoors committed:
add graphics

moravveji committed:
Baptizing mindwell (#592)

smoors committed:
update

smoors committed:
update

lexming committed:
Merge pull request #607 from JarneRenders/sofia-pilot

Add preliminary docs for sofia pilot phase

Commit:
remove irods-setup command

This command was deprecated shortly after the introduction of
PAM-interactive authentication.
Users should use iron instead.

jefscheepers committed:
Merge pull request #608 from hpcleuven/feature/remove_irods_setup

remove irods-setup command

MaximeVdB committed:
Quickstart for Mindwell (#609)

* Polish the hardware intro paragraphs

* Add a quick start guide for Mindwell

* Repeat backup and quota values for GPFS/Mindwell

* Clarify that GPFS is (not) yet accessible on Genius

* Clarify that there is no GPFS endpoint on Globus yet

* Add that interactive partitions are free of charge

Commit:
Add info about tmp on login nodes

Commit:
Include info about worker nodes

Commit:
Rewrite storage paragraphs on tmp storage

lexming committed:
Merge pull request #610 from JarneRenders/sofia-tmp

Add info about tmp on sofia login nodes

stevenvdb committed:
Retire NX (#611)

* Retire NX

* Remove NX entry from redirects.list

smoors committed:
add sofia job submission info

smoors committed:
update

smoors committed:
fix

smoors committed:
readd

smoors committed:
fix typo

Co-authored-by: Jarne Renders <jarne.renders@gmail.com>

lexming committed:
Merge pull request #612 from smoors/sofia-jobs

add sofia specific job submission info

MaximeVdB committed:
Update Leuven GPFS availability (#613)

GPFS storage is now available on the login nodes
and through Globus.

smoors committed:
sofia: add docs on job memory allocation

smoors committed:
update status note

stevenvdb committed:
Update MFA page so it refers to the new location access restriction section (#591)

* Update MFA page so it refers to the new location access restrictions

* Add note about access restrictions to page about login nodes

* Fix merge conflicts

lexming committed:
Merge pull request #615 from smoors/sofia-mem

sofia: add docs on job memory allocation

joachimbovin committed:
Added documentation to download metadata from Tier-1 Data as JSON

smoors committed:
add cpus-per-gpu docs for sofia

smoors committed:
update sofia support email address

lexming committed:
Merge pull request #619 from smoors/sofia-email

update sofia support email address

MaximeVdB committed:
Update Mindwell status (open pilot phase) (#620)

MaximeVdB committed:
Fix Mindwell production date (#621)

MaximeVdB committed:
Decommissioning of Genius CPU partitions (#531)

* Retire Genius Superdome

* Archive the next round of decommissioned Genius hardware

* Address remaining refs to decommissioned Genius parts

Moved some Genius quickstart content to wICE since that cluster will now be
the place new users will start out with.

Some oblique references remain in Torque-related pages, but I didn't change
those since we'll end up removing them after Hortense is gone.

* Add wICE and Mindwell in list of clusters with GPUs

mstfdkmn committed:
Add Tier-1 Data SFTP clients

mstfdkmn committed:
Fix typos reported by Mariana

stevenvdb committed:
Set up KU Leuven/UHasselt page about multiple parallel file systems (#624)

* Set up kuluh page about multiple parallel file systems

* Address comments on parallel file systems page

smoors committed:
Merge branch 'main' into cpus-per-gpu

smoors committed:
update

smoors committed:
update page structure

lexming committed:
Merge pull request #618 from smoors/cpus-per-gpu

add cpus-per-gpu docs for sofia

smoors committed:
fix sofia image alignment

joachimbovin committed:
Merge pull request #617 from hpcleuven/feature/tier1data_downloading_metadata

Documentation: metadata download Tier-1 Data

moravveji committed:
Jupyterlab updates (#589)

* PRC updates for OOD JupyterLab at KUL
* Bash kernel logo
* using resource usage plugin

stevenvdb committed:
Update note about turbo frequencies on Rocky 8 (#616)

moravveji committed:
Mindwll in (leuven) docs (#625)

revising mentions of Genius and adding Mindwell where needed

smoors committed:
sofia: add example gpu job resource request

lexming committed:
Merge pull request #627 from smoors/gpu-jobs

sofia: add example gpu job resource request

mstfdkmn committed:
Merge pull request #622 from hpcleuven/feature/Tier1Data-sftp-clients

Add Tier-1 Data SFTP clients

jooghe committed:
Update charge rates for Mindwell CPU and GPU resources (#628)

smoors committed:
update status msg for sofia

lexming committed:
Merge pull request #629 from smoors/no-pilot

update status msg for sofia

smoors committed:
use updated sofia logo for dark mode

lexming committed:
Merge pull request #630 from smoors/sofia-dark

use updated sofia logo for dark mode

WouterVanAssche committed:
updated section on "activating and using a project"

WouterVanAssche committed:
Merge pull request #521 from hpcleuven/rstudio_documentation_update

update vsc-Rproject documentation

smoors committed:
add globus endpoints for sofia

smoors committed:
update

smoors committed:
add links for globus + fix projects path

smoors committed:
update

lexming committed:
Merge pull request #631 from smoors/sofia-globus

add globus endpoints for sofia

Commit:
Add link to sofia portal

JarneRenders committed:
Change On-Demand to OnDemand

Co-authored-by: Sam Moors <smoors@users.noreply.github.com>

Commit:
Add basic sofia specific info to the portal docs

Commit:
Add sofia badge

Commit:
Add sofia tabs

Commit:
Add sofia tab everywhere it is relevant

smoors committed:
Merge pull request #633 from JarneRenders/sofia-portal

Add link to sofia portal

w-lampaert committed:
fix merge
